cmd/create/cluster/cmd.go

```diff
@@ -150,6 +153,23 @@ func init() {
 		"",
 	)
 	flags.MarkHidden("operator-iam-roles")
+
+	flags.StringVar(
+		&args.masterIAMCustomRole,
+		"master-iam-role",
+		"",
+		"Custom master IAM role to grant specific AWS permission",
```

Comment:
change to `"The name of the IAM role that will be attached to master instances"`